### PR TITLE
TT-1961: add support for multiple plugin bundles

### PR DESCRIPTION
**Pull request by imogenkraak**
## Description

```
Remove statement setting limit of one plugin file per bundle.
```

## Related Issue

[TT-1961](https://tyktech.atlassian.net/browse/TT-1961)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [ ] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Modules dependencies have been updated; run `go mod tidy` in applicable directories.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [x] `gofmt -s -w .`
  - [ ] `golangci-lint run`

[TT-1961]: https://tyktech.atlassian.net/browse/TT-1961?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: bundles/simple/plugin-2.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package main
```
Comment (umit) [inline bundles/simple/plugin-2.go:1]: @imogenkraak I would suggest renaming the file like this: `plugin_2.go`. And you can replace `plugin.go` with `plugin_1.go`.

File: bundles/simple/plugin-2.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+import (
+	"net/http"
+)
+
+// AddFooBarHeader2 adds custom "Foo: Bar" header to the request
+func AddFooBarHeader2(_ http.ResponseWriter, r *http.Request) {
+	r.Header.Add("Foo", "Bar")
+}
```
Comment (umit) [inline bundles/simple/plugin-2.go:10]: `AddFooBarHeader` function is already in `plugin.go`. We can write a new function like below.

```suggestion
// AddHelloWorldHeader adds custom "Hello: World" header to the request
func AddHelloWorldHeader(_ http.ResponseWriter, r *http.Request) {
	r.Header.Add("Hello", "World")
}
```

Comment (imogenkraak) [reply bundles/simple/plugin-2.go]: Haha ah yes I forgot to change. Have changed now :)

Comment (imogenkraak) [reply bundles/simple/plugin-2.go]: Haha ah yes I forgot to change. Have changed now :)

**Review by umit — APPROVED** [review]
LGTM

**Review by mitjaziv — APPROVED** [review]

**Pull request merged**